awx/main/management/commands/run_dispatcher.py

```diff
@@ -76,7 +76,7 @@ def handle(self, *arg, **options):
         consumer = None
 
         try:
-            queues = ['tower_broadcast_all', get_local_queuename()]
+            queues = ['tower_broadcast_all', 'tower_settings_change', 'rsyslog_configurer', get_task_queuename()]
```

Comment:
why is dispatcher listening to `rsyslog_configurer`

Reply:
to make this work the same i think we should remove rsyslog_configurer from dispatcher and in docker-compose devleopment environment's superviosr conf start the `awx-manage run_rsyslog_configurer.py`

Reply:
remove heartbeet

Reply:
fixed in #13768 13768

Reply:
We talked it over and I agree that removing `tower_settings_change` is what we need for right now - because it is consistent with all the 5 supervisor config files out there.

Reply:
https://github.com/ansible/awx/pull/13768